spec/features/fish/index_spec.rb

```diff
@@ -20,8 +20,14 @@
   end 
 
    scenario 'visitor sees link for fish index' do
-    visit "/aquariums/#{@shedd.id}"
+    visit "/fish"
 
     expect(page).to have_link("View Fish")
   end
+
+  scenario 'visitor sees link for aquarium index' do
+    visit "/fish"
+
+    expect(page).to have_link("View Aquariums")
```

Comment:
Could you test the path after the link has been clicked?

Reply:
Oh good call!